packages/core/src/notifications/controller.ts

```diff
@@ -42,7 +42,7 @@ export class NotificationsController {
 
     constructor(
         private readonly notificationsNode: NotificationsNode,
-        private readonly fetcher: NotificationFetcher = new RemoteFetcher()
+        private readonly fetcher: NotificationFetcher = new LocalFetcher()
```

Comment:
It's probably like this for testing but make sure to revert it when you are ready to merge.

Reply:
good point